[docs] Reduce the use of "our", "We"... (#17347)
mui-org/material-ui: docs/src/pages/guides/interoperability/interoperability.md

![npm](https://img.shields.io/npm/dm/react-jss.svg?)

Material-UI's styling solution shares many building blocks with [react-jss](https://github.com/cssinjs/react-jss).

> We went ahead and forked the project in order to handle Material-UI's unique needs, but we're working to merge the changes and fixes from Material-UI back to react-jss.

Author: ```suggestion
A fork was needed in order to handle Material-UI's unique needs, but with the intent to merge the changes and fixes from Material-UI back to react-jss.
```
